extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerImpl.java

```diff
@@ -1152,7 +1152,7 @@ private void makeShuffleOutputChannelFactory(boolean isFinalStage)
           );
     }
 
-    private <FactoryType extends FrameProcessorFactory<I, WorkerClass, T, R>, I, WorkerClass extends FrameProcessor<T>, T, R> void makeAndRunWorkProcessors()
+    private <FactoryType extends FrameProcessorFactory<T, R, I>, T, R, I> void makeAndRunWorkProcessors()
```

Comment:
Nit: this might be easier to read with more detailed generic names

Reply:
I updated the names to be clearer and added javadoc.